Wikipedia/Code/PlacesViewController.swift

```diff
@@ -52,13 +52,14 @@ class PlacesViewController: PreviewingViewController, UISearchBarDelegate, Artic
     @IBOutlet var emptySearchOverlayView: PlaceSearchEmptySearchOverlayView!
     
     @objc public var dataStore: MWKDataStore!
+    var articleGrouper: ArticleLocationGrouper!
 
     fileprivate let locationSearchFetcher = WMFLocationSearchFetcher()
     fileprivate let searchFetcher = WMFSearchFetcher()
     fileprivate let wikidataFetcher = WikidataFetcher()
     fileprivate let locationManager = WMFLocationManager.fine()
-    fileprivate let animationDuration = 0.6
-    fileprivate let animationScale = CGFloat(0.6)
+    internal let animationDuration = 0.6
```

Comment:
I suggest declaring a universal way of declaring Swift constants in the project.
In my [other PR](https://github.com/wikimedia/wikipedia-ios/pull/2136/files#diff-573fdbd6a3e8d237b1cd2e577595fed6R4) I suggested [this](clusteringViewContext.animationScale).